ament_mypy/ament_mypy/main.py

```diff
@@ -104,8 +104,10 @@ def main(argv: List[str] = sys.argv[1:]) -> int:
     print('\n{} files checked'.format(len(filenames)))
     if not normal_report:
         print('No errors found')
+        exit_code = 0
     else:
         print('{} errors'.format(len(errors_parsed)))
+        exit_code = len(errors_parsed)
```

Comment:
I'm not sure about this, because `errors_parsed` may not actually be errors. They could be "notes," which aren't really errors and shouldn't make the test exit with a non-zero exit code.

Reply:
Ah yes so my issue statement was wrong and the default behavior of mypy I think stems from this statement :

> Python will remain a dynamically typed language, and the authors have no desire to ever make type hints mandatory, even by convention

However I have a hard time seeing the purpose of mypy as an ament_package if it's not to enforce typing. As is, it is checking anything. I don't feel like it is coming with good default like other linters or formatters could have here. Would it make sense to update the default config file of mypy ?